### PR TITLE
Remove the in-cluster ceph part in rook ceph KB

### PR DESCRIPTION
**Pull request by futuretea**

### CODE REVIEW

File: kb/2023-08-23/using_rook_ceph_storage.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ One of the options for integrating external storage with Harvester is Rook, an o
 
 [Rook](https://rook.io) simplifies the deployment and management of Ceph, offering self-managing, self-scaling, and self-healing storage services. It leverages Kubernetes resources to automate the deployment, configuration, provisioning, scaling, upgrading, and monitoring of Ceph.
 
-In this article, we will walk you through the process of installing, configuring, and utilizing [Rook Ceph](https://rook.io/docs/rook/v1.12/Getting-Started/intro/) as a data disk for a VM within the Harvester environment.
+In this article, we will guide you through the process of installing, configuring, and utilizing [Rook](https://rook.io/docs/rook/v1.12/Getting-Started/intro/) to use storage from an [existing external Ceph cluster](https://www.rook.io/docs/rook/v1.12/CRDs/Cluster/external-cluster/) as a data disk for a VM within the Harvester environment.
```
Comment (guangbochen) [inline kb/2023-08-23/using_rook_ceph_storage.md:24]:
```suggestion
In this article, we will walk you through the process of installing, configuring, and utilizing [Rook](https://rook.io/docs/rook/v1.12/Getting-Started/intro/) from an [existing external Ceph cluster](https://www.rook.io/docs/rook/v1.12/CRDs/Cluster/external-cluster/) as a data disk for a VM within the Harvester environment.
```

**Review by guangbochen — APPROVED** [review]
Small nit, rest is LGTM

**Review by yasker — CHANGES_REQUESTED** [review]
Can we also rename the article to `Use Rook Ceph External Storage with Harvester`? Otherwise LGTM.

**Review by yasker — APPROVED** [review]

**Comment by yasker** [comment]
LGTM. Thanks.

**Pull request merged**